src/App.js

```diff
@@ -45,18 +45,6 @@ const INITIAL_DAO_STATE = {
   repos: [],
 }
 
-const SELECTOR_NETWORKS = [
```

Comment:
This felt like an odd place to configure the available networks during onboarding, so I've moved it out into the `/onboarding` folder as static configuration.